client/verta/verta/endpoint/_endpoint.py

```diff
@@ -186,22 +252,35 @@ def update(self, model_reference, strategy=None, wait=False, resources=None,
             Autoscaling condition for the updated endpoint.
         env_vars : dict of str to str, optional
             Environment variables.
+        kafka_settings : :class:`verta.endpoint.KafkaSettings` or False, optional
+            Kafka settings. If ``False``, clears this endpoint's existing Kafka
+            settings.
```

Comment:
The acceptance criteria mentions using ``None`` to deactivate the Kafka connection, but I personally find this to run against an implicit Python convention where ``None`` should mean "no parameter provided, do nothing", and would make it difficult for a user to say "update the model, but don't change or delete the current Kafka settings".

Reply:
I wish there was something more elegant, but I concur and like this as a solution.